thirdparty/vars-ubuntu.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+############################################################
+# You may have to set variables bellow,
+# which are used for compiling thirdparties and starrocks itself.
+############################################################
+
+# --job param for *make*
+PARALLEL=$[$(nproc)/4+1]
+
+###################################################
+# DO NOT change variables bellow unless you known
+# what you are doing.
+###################################################
+
+# thirdparties will be downloaded and unpacked here
+export TP_SOURCE_DIR=$TP_DIR/src
+
+# thirdparties will be installed to here
+export TP_INSTALL_DIR=$TP_DIR/installed
+
+# patches for all thirdparties
+export TP_PATCH_DIR=$TP_DIR/patches
+
+# header files of all thirdparties will be intalled to here
+export TP_INCLUDE_DIR=$TP_INSTALL_DIR/include
+
+# libraries of all thirdparties will be intalled to here
+export TP_LIB_DIR=$TP_INSTALL_DIR/lib
+
+# all java libraries will be unpacked to here
+export TP_JAR_DIR=$TP_INSTALL_DIR/lib/jar
+
+#####################################################
+# Download url, filename and unpacked filename
+# of all thirdparties
+#####################################################
+
+# libevent
+# the last release version of libevent is 2.1.8, which was released on 26 Jan 2017, that is too old.
+# so we use the master version of libevent, which is downloaded on 22 Jun 2018, with commit 24236aed01798303745470e6c498bf606e88724a
```

Comment:
I think we can upgrade the libevent and libbrpc to a higher version in StarRocks.

Reply:
I think upgrading libbrpc will certainly help. For libevent, I don't know if the `undefined reference: sysctl` error will be fixed by the upgrade.

Reply:
I will upgrade the brpc to 1.3.0. Only 1.3.0 have the above two prs.